### PR TITLE
javascript sdk release notes updated

### PR DESCRIPTION
**Pull request by jenniferopal**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \javascript sdk release notes updated
* Jira/GitHub Issue numbers (if any): \FFM-11825
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [X] Successful preview build.
- [X] Code owner review.
- [X] No merge conflicts.
- [X] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

**Review by erdirowlands — DISMISSED** [review]
Looks good

File: release-notes/feature-flags.md [hunk 1]
```diff
@@ -15,7 +15,18 @@ Review the notes below for details about recent changes to Harness Feature Flags
...
+## August 2024
+
+### Javascript SDK
+
+#### Version 1.26.3
+
+**Fixed issues**:
+ - The following CVEs have been patched:
+    - [ws(npm)](https://github.com/advisories/GHSA-3h5v-q93c-6h6q)
+    - [braces(npm)](https://github.com/advisories/GHSA-grv7-fg5c-xmjg)
```
Comment (erdirowlands) [inline release-notes/feature-flags.md:29]: question: What is the `npm` for?

Comment (erdirowlands) [reply release-notes/feature-flags.md:29]: Suggestion: it might be confusing, we can probably remove the `npm` part of the link.

Comment (jenniferopal) [reply release-notes/feature-flags.md:29]: No worries, I'll do that. Thank you. :)

**Review by erdirowlands — APPROVED** [review]

**Pull request merged**